test-engineering/contract-tests/volumes/client/scenarios.yml

```diff
@@ -3,6 +3,7 @@ scenarios:
     description: Test that Contile successfully returns tiles for Windows on Desktop.
     steps:
       - request:
+          service: 'contile'
```

Comment:
I don't think we need the quotes here.

Reply:
Change applied: [0f9761c](https://github.com/mozilla-services/contile/pull/422/commits/0f9761c7492a0a24605649d02542fa421fbda8da)

You are correct, we do not. All scenario and README files updated accordingly.